src/core/discrete_space.jl

```diff
@@ -354,12 +354,21 @@ function get_node_contents(coords::Tuple, model::ABM{A,<:DiscreteSpace}) where {
 end
 
 """
-    get_node_agents(x, model)
+    get_node_agents(x, model) → agents
+
 Same as `get_node_contents(x, model)` but directly returns the list of agents
 instead of just the list of IDs.
+
+    get_node_agents(x::Array{Any, 1}, model) → agents
```

Comment:
You can use `Vector{Any}` as a cleaner shorthand for `Array{Any, 1}`.

Reply:
Sure